vm.go

```diff
@@ -225,6 +261,23 @@ func (vm *VM) EvaluateSnippetMulti(filename string, snippet string) (files map[s
 	return
 }
 
+// EvaluateSnippetDependencies evaluates a string containing Jsonnet code returning an array of dependencies.
```

Comment:
It does not evaluate Jsonnet code. It finds all transitive dependencies (imports) of a Jsonnet program.

Reply:
Done.